src/test/java/seedu/simplykitchen/testutil/TypicalFood.java

```diff
@@ -0,0 +1,76 @@
+import static seedu.simplykitchen.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.simplykitchen.model.FoodInventory;
+import seedu.simplykitchen.model.food.Food;
+
+/**
+ * A utility class containing a list of {@code Food} objects to be used in tests.
+ */
+public class TypicalFood {
+
+    public static final Food ALICE = new FoodBuilder().withName("Alice Pauline")
+            .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
+            .withPhone("94351253")
+            .withTags("friends").build();
+    public static final Food BENSON = new FoodBuilder().withName("Benson Meier")
+            .withAddress("311, Clementi Ave 2, #02-25")
+            .withEmail("johnd@example.com").withPhone("98765432")
+            .withTags("owesMoney", "friends").build();
+    public static final Food CARL = new FoodBuilder().withName("Carl Kurz").withPhone("95352563")
+            .withEmail("heinz@example.com").withAddress("wall street").build();
+    public static final Food DANIEL = new FoodBuilder().withName("Daniel Meier").withPhone("87652533")
+            .withEmail("cornelia@example.com").withAddress("10th street").withTags("friends").build();
+    public static final Food ELLE = new FoodBuilder().withName("Elle Meyer").withPhone("9482224")
+            .withEmail("werner@example.com").withAddress("michegan ave").build();
+    public static final Food FIONA = new FoodBuilder().withName("Fiona Kunz").withPhone("9482427")
+            .withEmail("lydia@example.com").withAddress("little tokyo").build();
+    public static final Food GEORGE = new FoodBuilder().withName("George Best").withPhone("9482442")
+            .withEmail("anna@example.com").withAddress("4th street").build();
+
+    // Manually added
+    public static final Food HOON = new FoodBuilder().withName("Hoon Meier").withPhone("8482424")
+            .withEmail("stefan@example.com").withAddress("little india").build();
+    public static final Food IDA = new FoodBuilder().withName("Ida Mueller").withPhone("8482131")
+            .withEmail("hans@example.com").withAddress("chicago ave").build();
+
+    // Manually added - Food's details found in {@code CommandTestUtil}
+    public static final Food AMY = new FoodBuilder().withName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY)
+            .withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY).withTags(VALID_TAG_FRIEND).build();
+    public static final Food BOB = new FoodBuilder().withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
+            .withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
+            .build();
+
+    public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
+
+    private TypicalFood() {} // prevents instantiation
+
+    /**
+     * Returns an {@code FoodInventory} with all the typical persons.
```

Comment:
Extremely minor, but should "an" be "a"? Also should "persons" be "foods" instead?

Reply:
Thanks!